examples/ui.content/src/content/jcr_root/content/core-components-examples/library/form-hidden/.content.xml

```diff
@@ -0,0 +1,157 @@
+                    descriptionFromPage="false"
+                    fileReference="/content/dam/core-components-examples/library/github-logo.svg"
+                    linkURL="https://github.com/adobe/aem-core-wcm-components/tree/master/content/src/content/jcr_root/apps/core/wcm/components/form/hidden/v2/hidden"
+                    textIsRich="true"
+                    titleFromPage="false"/>
+                <teaser_556024830
+                    cq:styleIds="[1550165685463]"
+                    jcr:created="{Date}2019-02-14T16:41:59.595+01:00"
+                    jcr:createdBy="admin"
+                    jcr:description="&lt;p>Adobe Help Center&lt;/p>&#xd;&#xa;"
+                    jcr:lastModified="{Date}2019-12-19T16:58:10.548+01:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Using Core Components"
+                    sling:resourceType="core/wcm/components/teaser/v1/teaser"
+                    actionsEnabled="false"
+                    descriptionFromPage="false"
+                    fileReference="/content/dam/core-components-examples/library/adobe-logo.svg"
+                    linkURL="https://docs.adobe.com/content/help/en/experience-manager-core-components/using/components/forms/form-hidden.html"
+                    textIsRich="true"
+                    titleFromPage="false"/>
+                <title_865328580
+                    cq:styleIds="[1544759664489]"
+                    jcr:created="{Date}2018-12-06T19:22:23.729+01:00"
+                    jcr:createdBy="admin"
+                    jcr:lastModified="{Date}2018-12-10T17:08:09.903+01:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Examples"
+                    sling:resourceType="core/wcm/components/title/v2/title"
+                    type="h2"/>
+                <title
+                    cq:styleIds="[1544759676459]"
+                    jcr:created="{Date}2019-12-12T14:21:54.312+01:00"
+                    jcr:createdBy="admin"
+                    jcr:lastModified="{Date}2019-12-19T17:01:41.179+01:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Form Hidden field"
+                    sling:resourceType="core/wcm/components/title/v2/title"/>
+                <text_1910564414
+                    jcr:created="{Date}2019-12-12T14:22:11.663+01:00"
+                    jcr:createdBy="admin"
+                    jcr:lastModified="{Date}2019-12-19T17:01:31.312+01:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="core/wcm/components/text/v2/text"
+                    text="&lt;p>Form Container with a Hidden form fiel as child element.&amp;nbsp;&lt;/p>&#xd;&#xa;"
```

Comment:
same issue here, by wrapping it with the container, we don't see the hidden field markup, json and properties in the code section.

For the hidden field we also have an empty space where the component is usually rendered on the page, I wonder how we should deal with that. Perhaps we could hide the component rendering for that field, what do you think? It might be okay as it is?